### DISCUSSION TITLE
Voice-activated local chat w/ push-to-transmit

### DISCUSSION THREAD

**Discussion by James-E-A** [FakeFishGames/Barotrauma#9039] [Ideas and suggestions]
Right now, we have 3 options for voice chat control scheme:

1. Push-to-talk for radio _and_ push-to-talk local chat
   * (−) Have to maintain two keybinds
   * (−) PTT is cumbersome for casual conversations
2. Voice-activity radio; push-to-talk local
   * (−) Blasting your voice on a designated crosstalk channel 24/7
   * (−) PTT is cumbersome for casual conversations
3. No voice chat

All of these are annoying.

Why not have the option to locally-chat by default (voice activated) and **hold a radio key to transmit**?

**Comment by camille-david** [comment]
Hello James, not a dev, but I found out you can set the option manually.
All you have to do is : 
- go to the game files ;
- search for `config_player.xml` which is at the root of the game files ;
- set `uselocalvoicebydefault` to `true`.

Has to be done for each player. I hope it helps!

**Comment by JoneKone** [comment]
On a very crowded server Mic Spam is a big issue, I fully support this idea. To be forced from server side.
